### PR TITLE
docs: refine API docs for files in Search dir

### PR DESCRIPTION
**Pull request by devvaannsh**
`FindReplace.js` and `Search.js` files are removed from API docs as they are part of internal module.

### CODE REVIEW

File: src/search/FindInFiles.js [hunk 1]
```diff
@@ -381,6 +390,7 @@ define(function (require, exports, module) {
      * file exclusion filters, and isn't binary). Used when updating results incrementally - during the
      * initial search, these checks are done in bulk via getCandidateFiles() and the filterFileList() call
      * after it.
+     * @private
```
Comment (abose) [inline src/search/FindInFiles.js:393]: For readability, its best to add a single space between the description section in a JSDoc and the declaration/parameters/return section. Not in any currently done code, just a note for the future.

File: src/search/FindUtils.js [hunk 2]
```diff
@@ -35,10 +35,29 @@ define(function (require, exports, module) {
         StringUtils = require("utils/StringUtils"),
         _ = require("thirdparty/lodash");
 
-    var instantSearchDisabled = false,
-        indexingInProgress = false,
-        workerSearchCount = 0,
-        collapseResults = false;
+    /**
+     * if instant search is disabled, defaults to false
+     * @type {boolean}
```
Comment (abose) [inline src/search/FindUtils.js:40]: These variables are not exported and should be `@private` right?

File: src/search/QuickSearchField.js [hunk 3]
```diff
@@ -114,32 +114,59 @@ define(function (require, exports, module) {
...
+    /**
+     * @type {?jQueryObject} Dropdown's "ol", while open; null while closed
+     * @private
+     */
     QuickSearchField.prototype._$dropdown = null;
 
-    /** @type {!jQueryObject} */
+    /**
+     * @type {!jQueryObject}
+     * @private
```
Comment (abose) [inline src/search/QuickSearchField.js:155]: $something is not private

File: src/search/QuickSearchField.js [hunk 4]
```diff
@@ -114,32 +114,59 @@ define(function (require, exports, module) {
...
+     */
     QuickSearchField.prototype.$input = null;
 
-    /** @type {!jQueryObject} */
+    /**
+     * @type {!jQueryObject}
+     * @private
```
Comment (abose) [inline src/search/QuickSearchField.js:161]: $something is not private

File: src/search/ScrollTrackMarkers.js [hunk 5]
```diff
@@ -37,36 +37,42 @@ define(function (require, exports, module) {
     /**
```
Comment (abose) [inline src/search/ScrollTrackMarkers.js:37]: Remove this file from docs as this is a private API.

**Review by abose — CHANGES_REQUESTED** [review]

File: src/search/FindUtils.js (hunk 2 above)
Comment (devvaannsh) [reply src/search/FindUtils.js:40]: That's right. Missed it!

**Comment by abose** [comment]
LGTM Merging.
Thanks for contributing @devvaannsh

**Pull request merged**